docs/ReleaseNotes.txt

```diff
@@ -1,3 +1,131 @@
+=== v1.26.0 === (6 February 2025)
+
+ISPC release featuring improved ARM support, new "generic" targets that simplify
+ISPC's internal design and streamline the addition of new targets, improved code
+generation across x86 and ARM, and multiple stability fixes. This release is
+based on a patched LLVM 18.1.8.
+
+ARM Support Changes:
+
+ - The `--arch=arm` flag, which previously mapped to ARMv7 (32-bit), now maps to
+   ARMv8 (32-bit). There are no changes to `--arch=aarch64`, which continues to
+   map to ARMv8 (64-bit).
+ - The CPU definitions for the ARMv7 architecture have been removed: `cortex-a9`
+   and `cortex-a15`.
+ - New CPU definitions have been introduced, including `cortex-a55`,
+   `cortex-a78`, `cortex-a510`, and `cortex-a520`, along with support for new
+   Apple devices.
+ - New double-pumped targets have been introduced: `neon-i16x16` and
+   `neon-i8x32`.
+ - Dot product operations are now supported using native ARM instructions
+   (`sdot`/`udot`).
+ - Performance on ARMv8 has improved by an average on 13%.
+
+Generic Targets:
+
+In this release, generic targets were introduced in ISPC. Their main goal is to
+simplify ISPC target management and serve as the root implementation of
+internal builtins for hierarchical target linking. ARM targets were refactored
```

Comment:
"hierarchical target linking" I think it should be either explained better or removed. Release contes should be clear for the reader who don't have other context.